src/helper.js

```diff
@@ -126,6 +126,21 @@ export const toAmountLabel = (satoshis, settings) => {
     : formatNumber(toAmount(satoshis, settings.unit));
 };
 
+/**
+ * Convert a string formatted BTC amount either to fiat or the selected BTC unit.
+ * The output should be used throughout the UI for value labels.
+ * @param  {string} amount The amount e.g. '0.0001'
+ * @param  {Object} settings Contains the current exchange rate
+ * @return {string}          The corresponding value label
+ */
+export const toLabel = (amount, settings) => {
```

Comment:
Would it make more sense for `toAmountLabel` and `toLabel` to be named `satsToAmountLabel` and `btcToAmountLabel` respectively instead?

Reply:
> Would it make more sense for toAmountLabel and toLabel to be named satsToAmountLabel and btcToAmountLabel respectively instead?

Problem is the selected btc unit can be sat depending on the current settings.